### ISSUE TITLE
Inside Win-Unpacked Application is getting Default Electron Icon

### ISSUE THREAD

**Issue by Mrinanga** [electron-userland/electron-builder#8634]
* **Electron-Builder Version**: 

* **Node Version**: 

* **Electron Version**:
* **Electron Type (current, beta, nightly)**:

* **Target**:

**Comment by mmaietta** [comment]
This is correct ☝️ . `signAndEditExecutable: true` is what executes `rcedit.exe`, which is what applies the icon to the executable. I've investigated in the past if it's possible to split the rcedit.exe and signing logic, however, I'm not sure on the implications of doing so since they've always been executed together.
